python/ray/_private/metrics_agent.py

```diff
@@ -175,6 +212,7 @@ def record(self, metrics: List[Metric]):
         """
         self._last_reported_time = time.monotonic()
         for metric in metrics:
+            fix_metric(metric)
```

Comment:
Is this requiring high CPU usage? Should we have a simple check like statswith("grpc_") to avoid iterating every metrics?